docs/reference/ingest/processors/inference.asciidoc

```diff
@@ -116,3 +117,36 @@ categories for which the predicted probabilities are reported is 2
 (`num_top_classes`). The result is written to the `prediction` field and the top 
 classes to the `probabilities` field. Both fields are contained in the 
 `target_field` results object.
+
+
+[discrete]
+[[inference-processor-feature-importance]]
+==== {feat-imp-cap} object mapping
+
+Update your index mapping as you can see below to get the full benefit of 
+aggregating and searching for 
+{ml-docs}/dfa-classification.html#dfa-classification-feature-importance[{feat-imp}].
+
+[source,js]
+--------------------------------------------------
+{
+  "type": "nested",
+  "dynamic": true,
+  "properties": {
+    "feature_name": {
+      "type": "keyword"
+    },
+    "importance": {
+      "type": "double"
+    }
+  }
+}
+--------------------------------------------------
+// NOTCONSOLE
+
+The mapping field name for {feat-imp} is compounded as follows:
+
+`ml.<inference.target_field>`.`<inference.tag>`.`feature_importance`
+
+If `inference.tag` is not provided in the processor definition, it is not part 
+of the field path. The `<inference.target_field>` defaults to `inference`.
```

Comment:
probably some inference configs + field name examples?
```
{
    "tag": "foo",
    ...
}
```
Writes feature importance to field `ml.inference.foo.feature_importance`.

```
{
    "tag": "foo",
    "target_field": "my_field"
}
```
Writes feature importance to field `my_field.foo.feature_importance`.